Packs/Fortanix-DSM/Integrations/FortanixDSM/FortanixDSM.py

```diff
@@ -0,0 +1,579 @@
+    headers = headers if headers is not None else get_headers()
+
+    url = '{}/{}'.format(SERVER_URL, path)
+
+    res = requests.request(method, url, headers=headers, cert=get_client_cert(), data=data, params=params, verify=VERIFY_SSL)
+
+    if res.status_code < 200 or res.status_code >= 300:
+        try:
+            error_body = res.json()
+            if 'errors' in error_body and isinstance(error_body['errors'], list):
+                error_body = ';'.join(error_body['errors']) if len(error_body['errors']) > 0 else 'None'
+        except Exception as ex:
+            demisto.error("Error in send_request (parsing error msg): {}".format(ex))
+            error_body = res.content
+
+        raise ValueError('Request failed. Status code: {}, details: {}'.format(str(res.status_code), error_body))
+
+    if res.content:
+        return res.json()
+    return ''
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_command(cmd):
+    path = 'sys/v1/health'
+    if cmd == 'fortanix-test':
+        path = 'sys/v1/version'
+    res = send_request(path)
+    if cmd == 'fortanix-test':
+        readable_output = tableToMarkdown(f'Fortanix DSM status {res}', res)
+        return CommandResults(
+            outputs_prefix='Fortanix.DSM',
+            readable_output=readable_output,
+            outputs=res
+        )
+    else:
+        demisto.results('ok')
+
+
+def list_secrets_command():
+    group_id = demisto.args()['group_id'] if 'group_id' in demisto.args() else None
+    name = demisto.args()['name'] if 'name' in demisto.args() else None
+    kid = demisto.args()['kid'] if 'kid' in demisto.args() else None
+    state = demisto.args()['state'] if 'state' in demisto.args() else None
+    page = int(demisto.args()['page']) if 'page' in demisto.args() else None
+
+    path = 'crypto/v1/keys'
+    params = {
+        'obj_type': 'SECRET',
+        'sort': 'name:asc'
+    }
+    if state:
+        if state == 'enabled':
+            state = True
+        if state == 'disabled':
+            state = False
+        elif state == 'deleted':
+            params['show_deleted'] = 'true'
+        elif state == 'destroyed':
+            params['show_destroyed'] = 'true'
+
+    if page and page > 1:
+        params['offset'] = str((page - 1) * 100 + 1)
+    if kid:
+        path = path + '/' + kid
+    elif name:
+        params['name'] = name
+    else:
+        if group_id:
+            params['group_id'] = group_id
+        elif GROUP_IDS and len(GROUP_IDS) and GROUP_IDS.find(' ') < 0 and GROUP_IDS.find(',') < 0:
+            params['group_id'] = GROUP_IDS
+
+    res = send_request(path, 'get', params=params)
+    if not res:
+        raise ValueError('Secrets not found')
+
+    if kid and res:
+        readable_output = tableToMarkdown(f'Fortanix DSM Secret {res}', res)
+        return CommandResults(
+            outputs_prefix='Fortanix.Secret',
+            readable_output=readable_output,
+            outputs=res
+        )
```

Comment:
Add the raw response to all commandResults in all other functions:
```suggestion
        return CommandResults(
            outputs_prefix='Fortanix.Secret',
            readable_output=readable_output,
            outputs=res,
            raw_response=res
        )
```